docs/core/compatibility/windows-forms/8.0/obsolete-apis.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Breaking change: .NET 8 obsoletions and warnings in Windows Forms"
+titleSuffix: ""
+description: Learn about the .NET 8 breaking change where some Windows Forms APIs have been marked as obsolete or otherwise produce a warning.
+ms.date: 03/13/2023
+---
+# Windows Forms obsoletions and warnings (.NET 8)
+
+Some Windows Forms APIs have been marked as obsolete, starting in .NET 8.
+
+## Previous behavior
+
+In previous .NET versions, these APIs can be used without any build warning.
+
+## New behavior
+
+In .NET 8 and later versions, use of these APIs produces a compile-time warning or error with a custom diagnostic ID. The use of custom diagnostic IDs allows you to suppress the warnings individually instead of blanket-suppressing all obsoletion warnings.
+
+The following table lists the custom diagnostic IDs and their corresponding warning messages.
+
+| Diagnostic ID | Description | Severity | Version introduced |
+| - | - |
+| [WFDEV004](/dotnet/desktop/winforms/wfdev-diagnostics/wfdev004) | `ListBox.DefaultItemHeight` constant is no longer used as the default item height. Default item height is now scaled according to the application default font. | Warning | Preview 2 |
```

Comment:
@gewarren, this part of the change is not merged yet, it will go into the next preview.